src/components/Class/ClassCard/BatchCard.js

```diff
@@ -0,0 +1,473 @@
+            "
+              style={{ backgroundColor: "transparent" }}
+            >
+              Enrolled
+            </i>
+          )}
+          {((rolesList.length === 0 && item.enrolled) ||
+            (rolesList.length >= 1 &&
+              (item?.facilitator?.email === user.data.user.email || flag))) && (
+            <Typography>
+              <EditClass
+                item={item}
+                editClass={editClass}
+                indicator={true}
+                setRefreshKey={setRefreshKey}
+              />
+            </Typography>
+          )}
+        </Typography>
+        {showClass ? (
+          <Typography variant="body1" sx={{ display: "flex" }}>
+            <img
+              className={classes.icons}
+              src={require("../assets/calendar.svg")}
+            />
+            {format(item?.barch_start, "dd MMM yy")} -{" "}
+            {format(item?.batch_end, "dd MMM yy")}
+          </Typography>
+        ) : (
+          <Typography variant="body1" sx={{ display: "flex" }}>
+            <img
+              className={classes.icons}
+              src={require("../assets/calendar.svg")}
+            />
+            {format(item?.start_time, "dd MMM yy")} -{" "}
+            {format(item?.end_time, "dd MMM yy")}
+          </Typography>
+        )}
+
+        {/* <Typography variant="body1" sx={{ display: "flex" }}>
+          <img className={classes.icons} src={require("../assets/time.svg")} />
+          {format(classStartTime, "hh:mm aaa")} -{" "}
+          {format(classEndTime, "hh:mm aaa")}
+        </Typography> */}
+        <Typography variant="body1" sx={{ display: "flex" }}>
+          <img
+            className={classes.icons}
+            src={require("../assets/facilitator.svg")}
+          />
+          {item?.volunteer?.name || item?.facilitator?.name}
+        </Typography>
+        <Typography variant="body1" sx={{ display: "flex" }}>
+          <img
+            className={classes.icons}
+            src={require("../assets/language.svg")}
+          />
+          {languageMap[item?.lang]}
+        </Typography>
+
+        {/* it's for enroll class, join class and  class Timer button */}
+        <CardActions className={classes.cardActions}>
+          {item?.enrolled ? (
+            loading ? (
+              <div className="loader-button">
+                <Loader />
+              </div>
+            ) : (
+              <ClassJoinTimerButton
+                startTime={item?.start_time}
+                link={item?.meet_link}
+                onCanJoin={setCanJoin}
+              />
+            )
+          ) : loading ? (
+            <div className="loader-button">
+              <Loader />
+            </div>
+          ) : (
+            <Button
+              type="submit"
+              variant="text"
+              onClick={(e) => {
+                e.stopPropagation();
+
+                handleClickOpenEnroll(item.id);
```

Comment:
`handleClickOpenEnroll` doesn't take any inputs
```suggestion
                handleClickOpenEnroll();
```